app/assets/javascripts/classes/ConnectionModel.ts

```diff
@@ -66,20 +66,26 @@ class ConnectionModel {
     }
 
     private resetConnectionList() {
-        this.connectionList = <{path: String; protocol: Protocol}[]>[
-            {path: "/telemetry/v2/stream", protocol: new V2Protocol(this)},
-            {path: "/telemetry/v1/stream", protocol: new V1Protocol(this)},
-            {path: "/stream", protocol: new V1Protocol(this)}
-        ];
-    }
-
-    buildWebSocket() {
         var serverNameComponents = this.server.split(":");
         var serverHost = serverNameComponents[0];
         var serverPort = (typeof serverNameComponents[1] === "undefined") ? "7090" : serverNameComponents[1];
 
+        var directRoutePrefix = "ws://" + serverHost + ":" + serverPort;
```

Comment:
Since you're in here, you should match the protocol of the window.location so that we won't get ssl vs non-ssl errors.  Chrome is quite strict about that these days.